Lib/multi.ahk

```diff
@@ -1,14 +1,23 @@
 multi_ParseParameters() {
     global 0
-    global SNR_Array
-    Loop, %0% ;
+    ; global SNR_Array
+    ; Loop, %0% ;
+    ; {
+    ;     param := global %A_Index%
+    ;     SNR_Array[%A_Index%] := param
+    ; }
+
+    if 0 > 1
+    {
+        logger_Log("Too many parameters, use only 1")
```

Comment:
Only a single parameter (the test case name) is needed
as in [a relative link]MT_Auto.bat